src/main/java/frc/robot/robotcontainers/Robot2025Container.java

```diff
@@ -145,7 +146,9 @@ private void configureButtonBindings() {
     // operator.x().onTrue(new SetArm(()->ArmSetPoints.SPEAKER_KICKBOT_SHOT.angleDeg));
     // Climber
     //operator.leftTrigger(0.10).and(operator.back()).whileTrue(new ClimberRPM(()-> MathUtil.applyDeadband(operator.getLeftTriggerAxis(), 0.35) * 0.5));
-  
+    //Manipulator
```

Comment:
Please remove these new lines from Robot2025Container. We will add them here later after we have the competition robot.